package com.syntax.class34;

public class TestException {
	public static void main(String[] args) {
		
		TryCatchFinally.division(10,5);
		TryCatchFinally.division(10,0);
		
		System.out.println("*******END******");
		TryCatchFinally.read("");
	}
}
